roles/setup/handlers/main.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: Restart Receptor
+  ansible.builtin.systemd:
+    name: receptor
+    state: restarted
+    enabled: true
+  register: _restart
+
+- name: Reload Receptor
+  ansible.builtin.command: receptorctl --socket {{ receptor_socket_dir }}/{{ receptor_control_filename }} reload
+  become: true
```

Comment:
Can you not run the reload as whatever user you already are? Also, if become is already `true` (and not override-able) you don't need it here as well.